doc/manual/expressions/builtins.xml

```diff
@@ -1023,10 +1023,17 @@ in foo</programlisting>
 
     <listitem><para>Convert the expression
     <replaceable>e</replaceable> to a string.
-    <replaceable>e</replaceable> can be a string (in which case
-    <function>toString</function> is a no-op), a path (e.g.,
-    <literal>toString /foo/bar</literal> yields
-    <literal>"/foo/bar"</literal> or a set containing <literal>{ __toString = self: ...; }</literal>.</para></listitem>
+    <replaceable>e</replaceable> can be:
+    <itemizedlist>
+        <listitem>a string (in which case <function>toString</function> is a no-op)</listitem>
```

Comment:
My preference would be for "is the identity" instead of "is a no-op". Nix is a pure, functional language after all :)